internal/tool/tool.go

```diff
@@ -212,7 +223,7 @@ func filterIssuesFromKnownFiles(issues []codacy.Issue, knownFiles []string) []co
 // If the line number is not available in the Trivy result, try to find it in the source file.
 // Returns 0 if the line number is not found.
 func fallbackSearchForLineNumber(sourceDir, fileName, pkgName string) (int, error) {
-	filePath := path.Join(sourceDir, fileName)
+	filePath := filepath.Join(sourceDir, fileName)
```

Comment:
Is this to be Windows safe?
Change line 290 also then? And then remove the `path` require?

Reply:
according to Codacy, this is the OS safe way of doing it

Reply:
Codacy is wise. 😄